ext/scripts/install_scripts/install_via_conda.pl

```diff
@@ -0,0 +1,153 @@
+    --allow-no-version                    If --with-versions is active, allow packages to have no version specified
+    --allow-no-version-for-urls           If --with-versions is active, allow packages specified by urls to have no version
+    --ignore-installed                    Set the --ignore-installed option for pip
+    --use-deprecated-legacy-resolver      Set the --use-deprecated=legacy-resolver option for pip
+    --python-binary                       Python-binary to use for the installation
+                                     
+=cut
+
+use strict;
+use File::Basename;
+use lib dirname (__FILE__);
+use package_mgmt_utils;
+use Getopt::Long;
+
+my $help = 0;
+my $dry_run = 0;
+my $file = '';
+my $channel_file = '';
+my $conda_binary = '';
+my $with_versions = 0;
+my $allow_no_version = 0;
+GetOptions (
+            "help" => \$help,
+            "dry-run" => \$dry_run,
+            "file=s" => \$file,
+	    "channel-file=s" => \$channel_file,
+            "with-versions" => \$with_versions,
+            "allow-no-version" => \$allow_no_version,
+            "conda-binary=s" => \$conda_binary
+          ) or package_mgmt_utils::print_usage_and_abort(__FILE__,"Error in command line arguments",2);
+package_mgmt_utils::print_usage_and_abort(__FILE__,"",0) if $help;
+
+
+if($file eq ''){
+    package_mgmt_utils::print_usage_and_abort(__FILE__,"Error in command line arguments: --file was not specified",1);
+}
+
+if($channel_file eq ''){
+    package_mgmt_utils::print_usage_and_abort(__FILE__,"Error in command line arguments: --channel-file was not specified",1);
+}
+
+sub identity {
+    my ($line) = @_;
+    return $line 
+}
+
+sub replace_missing_version{
+    my ($line) = @_;
+    $line =~ s/==<<<<1>>>>//g;
+    return $line;
+}
+
+sub replace_line_for_missing_version{
+    my ($line) = @_;
+    $line =~ s/^.*==<<<<1>>>>$//g;
+    return $line;
+}
+
+sub generate_pinned_file{
+    my ($file, $dry_run, $with_versions, $allow_no_version) = @_;
+    my $element_separator = '\\|';
+    my $combining_template = "<<<<0>>>>";
+    my @templates=("'<<<<0>>>> ==<<<<1>>>>'");
+    my @separators = ("\n");
+
+    my @rendered_line_transformation_functions = (\&identity);
+    if($with_versions and $allow_no_version){
+        @rendered_line_transformation_functions = (\&replace_line_for_missing_version);
+    }
+
+    my $pinned_packages_file = 
+        package_mgmt_utils::generate_joined_and_transformed_string_from_file(
+            $file, $element_separator, $combining_template, \@templates, \@separators, \@rendered_line_transformation_functions);
+
+    if($with_versions){
+        if (index($pinned_packages_file, "==<<<<1>>>>") != -1) {
+            die "Pinned package file contains packages with unspecified versions, please check the package file '$file' or specifiy --allow-no-version. \n $pinned_packages_file";
+        } 
+    }
+    my $filename = '/opt/conda/conda-meta/pinned';
+    if($dry_run ne 0){
```

Comment:
looks strange, shouldn't be
```suggestion
    if($dry_run == 0){
```
?

Reply:
Done